src/Dnsimple/Client.php

```diff
@@ -22,21 +43,86 @@ class Client
     const API_VERSION = "v2";
 
     /**
-     * The default base URL for API requests.
+     * URL to the production environment
      */
-    const BASE_URL = "https://api.sandbox.dnsimple.com";
+    const BASE_URL = "https://api.dnsimple.com";
+    const DEFAULT_USER_AGENT = "dnsimple-php/" . VERSION;
 
 
     /**
      * @var string $accessToken the bearer authentication token
      */
-    private $accessToken;
+    private string $accessToken;
+
+    /**
+     * The http client we are using to send requests to the DNSimple API.
+     *
+     * @var GuzzleHttp\Client $httpClient the HTTP client
+     */
+    private GuzzleHttp\Client $httpClient;
 
     /**
-     * @var \GuzzleHttp\Client $httpClient the HTTP client
+     * @var Identity The service handling the Identity API
+     */
+    public Identity $Identity;
```

Comment:
Is this spelling intentional? AFAIK PHP has camelCase, so all services should be starting with lowercase letter.

```suggestion
    public Identity $identity;
```

Reply:
True, meaCulpa! 👼